bibangamba/react-fast-food-fast
# #163720144 Set up the react router

#### What does this PR do?
This PR delivers a chore to set up routing

#### Description of the tasks to be completed?
* install react-router-dom
* configure webpack to enable javascript to change URLs i.e. set historyApiFallback to true

#### How can this be manually tested?
- Clone the repo, checkout the branch and set up the app (`yarn install`)
- navigate to any random route, e.g. `localhost:3000/random`, it should 
show a page with a message saying the route was not found

#### Related pivotal tracker stories?
#163720144
